yarn.lock

```diff
@@ -11088,7 +11088,7 @@ rollup-plugin-livereload@^2.0.5:
   dependencies:
     livereload "^0.9.1"
 
-rollup-plugin-serve@^2.0.0:
+rollup-plugin-serve@^2.0.1:
```

Comment:
Why is this changing?

Reply:
I'm not sure why this was updated, I ran `yarn` and it updated the `yarn.lock` file.  I've reverted it as there have been no changes to the `package.json` file dependencies.